scripts/map-randomizer.js

```diff
@@ -0,0 +1,99 @@
+/**
+ *  Randomize array in-place using Durstenfeld shuffle algorithm.
+ *
+ * @public
```

Comment:
As we're following JSDoc for API documentation purposes, I would recommend adding a `@param {array} array <description of whatever the array is>` to this comment block, and flag it's a `@private` for a private API